dhall/src/Dhall/Repl.hs

```diff
@@ -4,6 +4,7 @@
 {-# language NamedFieldPuns    #-}
 {-# language OverloadedStrings #-}
 {-# language RecordWildCards   #-}
+{-# language ScopedTypeVariables   #-}
```

Comment:
The rest of the language extensions look like they're aligned. This is most likely done using `stylish-haskell`, it might be worth a try! :)

Reply:
Whoops I can delete this actually. Long story...